src/neptune/internal/utils/uncaught_exception_handler.py

```diff
@@ -76,4 +74,10 @@ def unregister(self, uid: uuid.UUID):
                 del self._handlers[uid]
 
 
+def exception_handler(uncaught_exception_handler: UncaughtExceptionHandler, exc_type, exc_val, exc_tb):
```

Comment:
- Instead of `exc_type, exc_val, exc_tb` let's use `*args` and `**kwargs`
- Let's make it a class method instead, you'll get `self` without a need to call `partial` and an access to internal property `_previous_uncaught_exception_handler` in a more proper way. Previously we had an inner function, right?